packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -202,10 +178,10 @@ export default [
           name: state.name,
           mode: state.isDelta ? 'delta' : 'full',
           compression: state.compression ? 'native' : '',
-          schedules: getNewSchedules(state.newSchedules),
+          schedules: normaliseSchedules(cloneDeep(state.schedules)),
           settings: {
-            ...getNewSettings({
-              schedules: state.newSchedules,
+            ...normaliseSettings({
+              settings: cloneDeep(state.settings),
```

Comment:
Why do you need to clone this?

Reply:
Because the function `normalizeSettings` mutate the settings' properties in deep.

Reply:
I see, my bad :slightly_smiling_face: